packages/router/src/lib/routes.ts

```diff
@@ -162,6 +184,7 @@ function toRoutes(rawRoutes: RawRoute[], files: Files): Route[] {
                 component: m.default,
                 ...toRouteConfig(m.routeMeta as RouteMeta | undefined),
                 children,
+                meta,
```

Comment:
To avoid confusion, I'd call this property `loadMeta` since we already use `meta` for meta tags.

Also, we can use a symbol as a key, to ensure uniqueness:

```ts
export const ROUTE_LOAD_META_KEY = Symbol(
  '@analogjs/router Route Load Meta Key'
);
```

Reply:
**EDIT:** Maybe `serverMeta` would be a more appropriate name 👀

Reply:
I named it something generic in case we want to store more metadata here in the future. Maybe `analogMeta` or `configMeta`?